### ISSUE TITLE
[RN 0.67.3] Plugin with id 'maven' not found. 

### ISSUE THREAD

**Issue by alz10** [ammarahm-ed/react-native-admob-native-ads#269]
Any update to support `RN v0.67+`
```
1: Task failed with an exception.
-----------
* Where:
Build file '/Users/user/Desktop/testapp (v0.67.3) - (v1)/testapp/node_modules/react-native-admob-native-ads/android/build.gradle' line: 31

* What went wrong:
A problem occurred evaluating project ':react-native-admob-native-ads'.
> Plugin with id 'maven' not found.

==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring project ':react-native-admob-native-ads'.
> com.android.builder.errors.EvalIssueException: compileSdkVersion is not specified. Please add it to build.gradle

```

dependencies
```
"react": "17.0.2",
"react-native": "0.67.3",
"react-native-admob-native-ads": "^0.5.0",
```

**Comment by alz10** [comment]
I had to downgrade gradle to 6.9

Folder: 
`android/gradle/wrapper/gradle-wrapper.properties`

Change 7.2 to 6.9 if you're using `RN 0.67+`: 
`distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip`

**Comment by blazerroadg** [comment]
work around solution
on file: 
/node_modules/react-native-admob-native-ads/android/build.gradle
change line 31 to:
apply plugin: 'maven-publish'

and comment 139 to 143
  ```
// repositories.mavenDeployer {
        //     // Deploy to react-native-event-bridge/maven, ready to publish to npm
        //     repository url: "file://${projectDir}/../android/maven"
        //     configureReactNativePom pom
        // }


```

**Comment by SergiOnGit** [comment]
> work around solution on file: /node_modules/react-native-admob-native-ads/android/build.gradle change line 31 to: apply plugin: 'maven-publish'
> 
> and comment 139 to 143
> 
> ```
> // repositories.mavenDeployer {
>       //     // Deploy to react-native-event-bridge/maven, ready to publish to npm
>       //     repository url: "file://${projectDir}/../android/maven"
>       //     configureReactNativePom pom
>       // }
> ```

I can confirm that this solution works. Please fix this. We can't use RN 0.67 without it @ammarahm-ed

**Comment by mhakashcw** [comment]
> work around solution on file: /node_modules/react-native-admob-native-ads/android/build.gradle change line 31 to: apply plugin: 'maven-publish'
> 
> and comment 139 to 143
> 
> ```
> // repositories.mavenDeployer {
>       //     // Deploy to react-native-event-bridge/maven, ready to publish to npm
>       //     repository url: "file://${projectDir}/../android/maven"
>       //     configureReactNativePom pom
>       // }
> ```

In my case I am now getting this error
```* Where:
Build file '/Users/akash/codes/react-native/v68/node_modules/react-native-admob-native-ads/android/build.gradle' line: 108

* What went wrong:
A problem occurred configuring project ':react-native-admob-native-ads'.
> Configuration with name 'compile' not found.```

**Comment by KevMac19** [comment]
> > work around solution on file: /node_modules/react-native-admob-native-ads/android/build.gradle change line 31 to: apply plugin: 'maven-publish'
> > and comment 139 to 143
> > ```
> > // repositories.mavenDeployer {
> >       //     // Deploy to react-native-event-bridge/maven, ready to publish to npm
> >       //     repository url: "file://${projectDir}/../android/maven"
> >       //     configureReactNativePom pom
> >       // }
> > ```
> 
> In my case I am now getting this error
> 
> ```
> Build file '/Users/akash/codes/react-native/v68/node_modules/react-native-admob-native-ads/android/build.gradle' line: 108
> 
> * What went wrong:
> A problem occurred configuring project ':react-native-admob-native-ads'.
> > Configuration with name 'compile' not found.```
> ```

Same error

**Comment by soundaraj** [comment]
> > > work around solution on file: /node_modules/react-native-admob-native-ads/android/build.gradle change line 31 to: apply plugin: 'maven-publish'
> > > and comment 139 to 143
> > > ```
> > > // repositories.mavenDeployer {
> > >       //     // Deploy to react-native-event-bridge/maven, ready to publish to npm
> > >       //     repository url: "file://${projectDir}/../android/maven"
> > >       //     configureReactNativePom pom
> > >       // }
> > > ```
> > 
> > 
> > In my case I am now getting this error
> > ```
> > Build file '/Users/akash/codes/react-native/v68/node_modules/react-native-admob-native-ads/android/build.gradle' line: 108
> > 
> > * What went wrong:
> > A problem occurred configuring project ':react-native-admob-native-ads'.
> > > Configuration with name 'compile' not found.```
> > ```
> 
> Same error

comment line 108 :)

**Comment by mhakashcw** [comment]
> > > > work around solution on file: /node_modules/react-native-admob-native-ads/android/build.gradle change line 31 to: apply plugin: 'maven-publish'
> > > > and comment 139 to 143
> > > > ```
> > > > // repositories.mavenDeployer {
> > > >       //     // Deploy to react-native-event-bridge/maven, ready to publish to npm
> > > >       //     repository url: "file://${projectDir}/../android/maven"
> > > >       //     configureReactNativePom pom
> > > >       // }
> > > > ```
> > > 
> > > 
> > > In my case I am now getting this error
> > > ```
> > > Build file '/Users/akash/codes/react-native/v68/node_modules/react-native-admob-native-ads/android/build.gradle' line: 108
> > > 
> > > * What went wrong:
> > > A problem occurred configuring project ':react-native-admob-native-ads'.
> > > > Configuration with name 'compile' not found.```
> > > ```
> > 
> > 
> > Same error
> 
> comment line 108 :)

In that case, the app doesn't even open it crashes

**Comment by soundaraj** [comment]
> > > > > work around solution on file: /node_modules/react-native-admob-native-ads/android/build.gradle change line 31 to: apply plugin: 'maven-publish'
> > > > > and comment 139 to 143
> > > > > ```
> > > > > // repositories.mavenDeployer {
> > > > >       //     // Deploy to react-native-event-bridge/maven, ready to publish to npm
> > > > >       //     repository url: "file://${projectDir}/../android/maven"
> > > > >       //     configureReactNativePom pom
> > > > >       // }
> > > > > ```
> > > > 
> > > > 
> > > > In my case I am now getting this error
> > > > ```
> > > > Build file '/Users/akash/codes/react-native/v68/node_modules/react-native-admob-native-ads/android/build.gradle' line: 108
> > > > 
> > > > * What went wrong:
> > > > A problem occurred configuring project ':react-native-admob-native-ads'.
> > > > > Configuration with name 'compile' not found.```
> > > > ```
> > > 
> > > 
> > > Same error
> > 
> > 
> > comment line 108 :)
> 
> In that case, the app doesn't even open it crashes

can you give me a app crash logs

**Comment by ammarahm-ed** [comment]
Also this issue is fixed on master already for v0.6.0

**Comment by ivan-mumtak** [comment]
> Also this issue is fixed on master already for v0.6.0
Can you please publish v0.6.0? It will be very helpfully to have this fix soon because now I am stuck with some library updates caused by maven
